experiment/builder_runner.py

```diff
@@ -905,12 +905,22 @@ def build_and_run_cloud(
         f'--upload_coverage={coverage_path}',
         f'--upload_reproducer={reproducer_path}',
         f'--upload_corpus={corpus_path}',
-        f'--experiment_name={self.experiment_name}'
+        f'--experiment_name={self.experiment_name}',
+        f'--real_project={project_name}',
     ]
+
+    if oss_fuzz_checkout.ENABLE_CACHING and (
+        oss_fuzz_checkout.is_image_cached(project_name, 'address') and
+        oss_fuzz_checkout.is_image_cached(project_name, 'coverage')):
```

Comment:
nit:
1. Do we need both `address` and `coverage` to use the cached image? E.g., I would consider using a project's cache even if it does not have `coverage`, because `address` image is the one that will be repeatedly used during fuzz target generation. Do you happen to know many projects support `address` but not `coverage`?
2. Misplaced `(`. I.e., `if (oss_fuzz_checkout.ENABLE_CACHING and ... and ...):`